doc/docs/resources.md

```diff
@@ -0,0 +1,4 @@
+# Additional Resources
+
+- [Blog Post about a lightweight service manager](https://www.redhat.com/en/blog/hirte-renamed-eclipse-bluechi)
```

Comment:
Could you please add also a link to DevConf 2023 presentation?

https://www.youtube.com/watch?v=8RiRiviSVqM

Reply:
Done :heavy_check_mark: